scripts/exgfs_tc_genesis.sh

```diff
@@ -40,13 +40,13 @@ mkdir -p $pertdir
 
 #outfile=${pertdir}/trkr.${regtype}.${cmodel}.${pert}.${ymdh}.out
 
-if [[ -d /scratch2 ]] ; then
-  # We are on NOAA Hera
+if [[ -d /scratch3 ]] ; then
+  # We are on NOAA Hera or Ursa
   machine=hera
```

Comment:
Should we just remove Hera here and replace it with Ursa? Otherwise this should be `ursa` or have separate blocks for each machine.

Reply:
Yeah, we can replace with ursa.

```suggestion
  machine=ursa
```